benchmarks/benchmarks.py

```diff
@@ -206,35 +206,35 @@ def time_tree_create(self):
         tree = pygeos.STRtree(self.polygons)
         tree.query(pygeos.points(0, 0))
 
-    def time_tree_query_bulk(self):
-        self.tree.query_bulk(self.polygons)
+    def time_tree_query(self):
```

Comment:
Benchmarks are untested; these changes are just to pick up the `query_bulk` => `query` transition.